sktime/utils/plotting.py

```diff
@@ -23,6 +23,8 @@ def plot_series(*series, labels=None, markers=None):
     markers: list, optional (default=None)
         Markers of data points, if None the marker "o" is used by default.
         Lenght of list has to match with number of series
+    pred_int: pd.DataFrame, optional (default=None)
```

Comment:
if we say that `pred_int` is a `pd.DataFrame`, we need to check that in the function and raise an error if not. 

We should define a helper function for this check and check for at least the following I believe: 
```python
def check_pred_int(pred_int):
    assert isinstance(pred_int, pd.DataFrame)
    assert pred_int.shape[1] == 2
    assert pred_int.columns == ["lower", "upper"]
```
Instead of using assertions, we should raise an ValueError in the code to give a more user-friendly error message. Does that help?